about.html

```diff
@@ -0,0 +1,171 @@
+          <span
+            ><a href="mailto:someone@example.com">nutirstion@gmai.com</a></span
+          >
+        </p>
+      </div>
+      <div class="row">
+        <section class="logo">
+          <h2 class="a">Creative Common<br />Nutrition 2016 Logo</h2>
+          <h2 class="b">Creative Common Nutrition 2016 Logo</h2>
+          <hr />
+          <p>
+            The logo of Nutirstion 2015 was decided <br />
+            through the logo competetion from 8.Jun to 7.July
+          </p>
+          <img src="./images/img4.png" alt="logo" />
+        </section>
+      </div>
+      <div class="row">
+        <section class="healthyRecipes">
+          <div>
+            <h2>See Easy and Cheap Healthy Recipes</h2>
+            <hr />
+            <p class="a">
+              Take a look at the easiest healty recipes which takes <br />
+              no time to prepare it .
+            </p>
+            <p class="b">
+              Take a look at the easiest healty recipes which takes no time to
+              prepare it .
+            </p>
+          </div>
+          <div class="container">
+            <div class="recipes1">
+              <div class="recipeImage">
+                <img
+                  src="./images/Coconut-and-corn-curry-3ef570b.jpg"
+                  alt="Coconuts"
+                  class="sepia"
+                />
+              </div>
+              <div class="centerd">
+                <h3>Coconut & corn curry</h3>
+                <p>
+                  <a href="recipe1.html" target="_blank"> See the Recipe</a>
+                </p>
+              </div>
+            </div>
+            <div class="recipes1">
+              <div class="recipeImage">
+                <img
+                  src="./images/Miso-and-butternut-soup-efe9277.jpg"
+                  alt="Miso"
+                  class="sepia"
+                />
+              </div>
+              <div class="centerd">
+                <h3>Miso & butternut soup</h3>
+                <p>
+                  <a href="recipe2.html" target="_blank">See the Recipe</a>
+                </p>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+      <div class="row">
+        <section class="partner a" id="partner">
+          <h3>Parnter</h3>
+          <hr />
+          <span class="mozilla"> mozilla </span>
+          <span class="google"> Google </span>
+          <span class="naver"> Naver </span>
+          <span class="daumkakao"> daumkakao </span>
+          <span class="airbnb"> airbnb </span>
+        </section>
+      </div>
+      <footer>
+        <div class="left-side">
+          <span> <img src="./images/cc.svg" /></span>
+          <p>Creative <br />Commons Korea</p>
+        </div>
+
+        <div class="right-side">
+          <p>2015 Creative Common Korrea . <br />Some rigths reserved</p>
+        </div>
+      </footer>
+    </div>
```

Comment:
- [ ] Kindly fix the alignment of the contents in your `footer`.
I think to make it simple you can just have your logo aligned to the left and text aligned to the right.
The same logo you used in the navigation bar.

### Your footer 
![Screenshot from 2022-10-25 18-32-13](https://user-images.githubusercontent.com/89228743/197853818-2c00bb36-535a-4623-b857-be4a96ecef0b.png)
 ### The template 
![Screenshot from 2022-10-25 18-33-42](https://user-images.githubusercontent.com/89228743/197854032-06e1700e-ce40-458e-bc52-8b5868eac40f.png)